packages/core/src/server/environment.ts

```diff
@@ -0,0 +1,76 @@
+import { join } from 'node:path';
+import type { EnvironmentContext, Stats } from '@rsbuild/shared';
+import { run } from './runner';
+
+export type ServerUtils = {
+  readFileSync: (fileName: string) => string;
+  environment: EnvironmentContext;
+};
+
+export const loadBundle = async <T>(
+  stats: Stats,
+  entryName: string,
+  utils: ServerUtils,
+): Promise<T> => {
+  const { chunks, entrypoints, outputPath } = stats.toJson({
+    all: false,
+    chunks: true,
+    entrypoints: true,
+    outputPath: true,
+  });
+
+  if (!entrypoints?.[entryName]) {
+    throw new Error(`can't find ssr entry(${entryName})`);
```

Comment:
This API can only be used with SSR? It there any other scenarios

Reply:
log problem, other scenarios are also supported.